### PR TITLE
Add sizing to tasks

### PR DESCRIPTION
**Pull request by zoeblairfriedman**

### CODE REVIEW

File: ' [hunk 1]
```diff
@@ -0,0 +1,18 @@
+- @current_action = controller.request.path_parameters[:action].to_sym
+
+h1
+  - if @board.present?
+    = @board.name.titleize
```
Comment (zoeblairfriedman) [inline ':5]: This is the only line I changed—not sure why this whole chunk appears in the PR.

Comment (m4thayus) [reply ':5]: This is in a file called `'` so it's a completely new file...

File: app/models/task.rb [hunk 2]
```diff
@@ -1,17 +1,26 @@
...
   validates :title, presence: true
+  validates :size, inclusion: { in: SIZES }, allow_nil: true
 
   scope :active, -> { where(completed_at: nil) }
   scope :recently_completed, -> { where("completed_at > ?", 1.week.ago) }
 
   private
 
+  def nillify_size
+    self.size = nil if size&.empty? # rubocop:disable Lint/SafeNavigationWithEmpty
```
Comment (m4thayus) [inline app/models/task.rb:21]: Don't use empty use blank. Empty is for array like things. Also it raises when called on `nil` blank doesn't so you don't need safe navigation. Rubocop is not wrong here

File: app/views/tasks/_table.html.slim [hunk 3]
```diff
@@ -2,19 +2,20 @@ table
...
         th
           = header.titleize
-    - if @board && @board&.name
-      th
-        ' Approved
-    - elsif !show_statistics
-      th
-        ' Reviewed
+    - else
+      - if @board && @board&.name
```
Comment (m4thayus) [inline app/views/tasks/_table.html.slim:13]: The `@board &&` is redundant. Just `@board&.name` is fine

**Review by m4thayus — CHANGES_REQUESTED** [review]

File: ' (hunk 1 above)
Comment (zoeblairfriedman) [reply ':5]: that is so weird. but yes, thank you—so sorry

Comment (m4thayus) [reply ']: no worries 😃

**Review by m4thayus — APPROVED** [review]

**Pull request merged**